configs/config.go

```diff
@@ -96,6 +96,9 @@ type Config struct {
 	// ReadonlyPaths specifies paths within the container's rootfs to remount as read-only
 	// so that these files prevent any writes.
 	ReadonlyPaths []string `json:"readonly_paths"`
+
+	// Container's standard descriptors (std{in,out,err}), needed for checkpoint and restore
+	StdFds [3]string `json:"ext_pipes,omitempty"`
```

Comment:
@mrunalp 
Sure, any suggestions?  The name ext_pipes was chosen because the standard file descriptors are actually external pipes but I am open to better suggestions.